script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -803,6 +808,8 @@ Besides, if you don't have to worry about dying from old age, shouldn't that be
 [sha:5f0ec949bc023b54b108aa22f4e28d190c75e063]{
 -- Page 568, Offset 17997.
 -- 「だから、それで十分になってしまったのよ、彼らは。
+// B: this line is a little confusing to me. If it's 'enough', why is she
+// saying their lives have no meaning on the next line?
```

Comment:
I read it as 'too much of a good thing', 'more than they bargained for' where living for a long time _should_ be great but wraps past it to being bad